tests/destructor/tnewruntime_misc.nim

```diff
@@ -1,5 +1,6 @@
 discard """
   cmd: '''nim cpp -d:nimAllocStats --newruntime --threads:on $file'''
+  disabled: "true"
```

Comment:
Can't you test this with `--gc:arc` instead?

Reply:
It doesn't work with `--gc:arc`, so I disabled it.